.travis.yml

```diff
@@ -42,9 +42,17 @@ script:
     --organism Mus_musculus
     --outdir tests/unit/format-ATtRACT-motifs/ATtRACT_mmu
   - md5sum --check tests/unit/format-ATtRACT-motifs/expected_output_mmu.md5
+  # Test parse.py script
+  - >
+    python scripts/parse.py
+    --indir tests/unit/combine_results/motif_HNRNPF_820 tests/unit/combine_results/motif_HNRNPF_821 tests/unit/combine_results/motif_HNRNPF_822 tests/unit/combine_results/motif_HNRNPF_823 tests/unit/combine_results/motif_HNRNPF_824
```

Comment:
could you please change the argument to `input-directories` explicite? :)

Reply:
Done!